sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -26,12 +31,10 @@ export class ConfirmationOfEnrollmentController {
    * @param locationId location that is completing the COE.
    * @returns student application list of an institution location
    */
-  @AllowAuthorizedParty(AuthorizedParties.institution)
```

Comment:
why is `AuthorizedParties.institution` removed

Reply:
It is was present at the controller level and at the method level and there is no need to keep both.
I kept it only at the controller level.

Reply:
got it 👍